doc/README.md

```diff
@@ -54,16 +52,15 @@ Simple examples for all features can be found in the
 [examples folder](https://github.com/flame-engine/flame/tree/main/examples).
 
 You can also check out the
-[awesome flame repository](https://github.com/flame-engine/awesome-flame#user-content-articles--tutorials),
-it contains quite a lot of good tutorials and articles written by the community to get you started
+[awesome flame repository](https://github.com/flame-engine/awesome-flame#user-content-articles--tutorials).
```

Comment:
It's better to declare this long link at the end (for example, see source of `documentation.md`)

Reply:
Ok, I am not seeing anything, but will look into this.  This is what was originally in the file.

Reply:
https://github.com/flame-engine/flame/blob/main/doc/development/documentation.md?plain=1#L1-L10
See how if the links are defined separately, the text is much easier to read.

Reply:
Oh my, ok, yes I saw that but I was looking somewhere entirely else thinking there was some form of special markdown.  Ok, yes and will do.

Reply:
Resolved.